developer-docs-site/docs/nodes/identity-and-configuration.md

```diff
@@ -0,0 +1,199 @@
+- validator-full-node-**identity**.yaml contains the private keys for validator fullnode and public fullnode. 
+
+Hence if you are looking for your machine’s private keys, look for YAML filenames with  `identity` in them.
+
+#### Configuration
+
+Machine **configuration** is also defined in a YAML file. A machine configuration YAML **never contains any key, public or private**. For example, the configuration YAMLs validator.yaml, fullnode.yaml, docker-compose.yaml and docker-compose-fullnode.yaml **do not contain any keys.** 
+
+As noted earlier, a machine has an identity and a configuration. Hence:
+
+- For a validator node, identity is defined in validator-**identity**.yaml and configuration is in validator.yaml. 
+- For a validator fullnode, its identity is defined in validator-full-node-**identity**.yaml and its configuration is defined in fullnode.yaml.
+
+### Persona
+
+#### Identity and configuration
+
+A persona has a single YAML that combines the persona’s identity and configuration information. For example, for the three personas, owner, operator and voter:
+
+- An owner's identity-configuration is defined in **owner.yaml**. The owner.yaml contains the public keys and blockchain account addresses for owner, operator and voter, and some owner-specific configuration such as stake amount and commission percentage. 
+- An operator’s identity-configuration is defined in **operator.yaml**. The operator.yaml contains public keys and blockchain account address for the operator and some machine configuration information plus a consensus public key and consensus proof of possession key. **Only the operator has the consensus keys.** Neither the owner nor the voter has the consensus keys. 
+- A voter's identity-configuration, i.e., voter.yaml, does not exist. 
+
+## Description of identity YAMLs
+
+This section explains the following key and identity YAML files that are generated during the deployment of a validator node:
+
+- `public-keys.yaml`.
+- `private-keys.yaml`.
+- `validator-identity.yaml`.
+- `validator-full-node-identity.yaml`.
+
+The following command is used to generate the above key and identity YAMLs. See, for example, [Step 10 while using AWS to deploy the validator node](validator-node/operator/running-validator-node/run-validator-node-using-aws), or in [Step 10 while using GCP](validator-node/operator/running-validator-node/run-validator-node-using-gcp). 
+
+```bash
+aptos genesis generate-keys --output-dir ~/$WORKSPACE/keys
+```
+
+See below a diagram that shows how the identities for the validator node and validator fullnode are derived from the private and public keys:
+
+<ThemedImage
+alt="Signed Transaction Flow"
+sources={{
+    light: useBaseUrl('/img/docs/key-yamls.svg'),
+    dark: useBaseUrl('/img/docs/key-yamls-dark.svg'),
+  }}
+/>
+
+### public-keys.yaml
+
+#### Example
+
+Click below to see an example YAML configuration:
+<details>
+<summary>public-keys.yaml</summary>
+
+```yaml
+---
```

Comment:
Do we want to note that this is the validator address (stake pool address or machine address whatever you want to consistently call it), and not the owner, operator, or voter address? cc @sherry-x. We can also link to the get-stake-pool CLI doc to explain how to get all of these addresses (from owner address) as well

Reply:
Yes. This is exactly the kind of review feedback I needed. I didn't understand how I should understand the account address here. With the above explanation, @movekevin, now I have a different question: how is this address different from a resource account address? I won't mention it, but I want to understand it. Yes to linking also.